src/chrome/chromeDebugAdapter.ts

```diff
@@ -400,6 +400,10 @@ export abstract class ChromeDebugAdapter implements IDebugAdapter {
         args.sourceMaps = typeof args.sourceMaps === 'undefined' || args.sourceMaps;
 
         this._smartStepEnabled = this._launchAttachArgs.smartStep;
+
+        // Filter down to a set of requested target types. Note that `targetTypes` might be `undefined` here, which is expected;
+        // the result will be disabling target type filtering unless explicitly requested by the consumer.
+        this._chromeConnection.setTargetTypes(args.targetTypes);
```

Comment:
Yeah, I think we could pull it off without much friction…we'd just need to expose a public method like this one, so that ChromeConnection, in addition to possibly receiving `targetFilter` at construction time, also can receive target filter at args-time. This block would roughly change to:

```
if (args.targetFilter) {
  this._chromeConnection.setTargetFilter(args.targetFilter);
}
```

You want me to go with that approach instead?

Reply:
(cc: @roblourens)

Reply:
Yeah, I think that makes sense.